CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 - Remove mongodb dependence from packages.json (already in iota-node-lib)
+- Add: allow NGSIv2 for updating active attributes at CB, throught configuration based on 
```

Comment:
The line has been cut... It should be:

```
 Add: allow NGSIv2 for updating active attributes at CB, throught configuration based on iotagent-node-lib(#233)
```

(This seems to happended in the last merge from master into PR branch)

Reply:
Fixed in fdd0abfc236dfdb391572e67b1e675785f3c4ce1